src/common/utxobased/db/makeProcessor.ts

```diff
@@ -182,20 +190,22 @@ export async function makeProcessor(
     },
 
     async getUsedAddress(scriptPubkey: string): Promise<boolean> {
+      const cleaner = asBoolean
       const [used] = await baselets.all.usedFlagByScriptPubkey.query('', [
         scriptPubkey
       ])
-      return used
+      return cleaner(used)
```

Comment:
Instead of what you did here, you should have something like this:
```
const doc = await baselets.all.usedFlagByScriptPubkey.query('', [
   scriptPubkey
])
return asUsed(doc) // This cleaner would verify the doc and return just the used if it's valid
```

This comment is relevant to almost all your usages of cleaners in this file